### PR TITLE
"Create an account" not clearly visible in dark mode #576

### PR DESCRIPTION
**Pull request by vivekBoii**
"Create an account" not clearly visible in dark mode #576

### CODE REVIEW

File: .env.example
Comment (devkiran) [inline .env.example:1]: Can you please revert this change? Why did you remove this file?

**Review by devkiran — CHANGES_REQUESTED** [review]

**Comment by vivekBoii** [comment]
restored it Sir

File: .env.example
Comment (vivekBoii) [reply .env.example:1]: resolved

**Comment by devkiran** [comment]
Can you please check it again? I can see the file `.env.example`is deleted

**Comment by devkiran** [comment]
Closing this in favor of https://github.com/boxyhq/saas-starter-kit/pull/584
